src/components/app.css

```diff
@@ -21,14 +21,21 @@
 
   /* Colors */
   --light-color: #dedede;
-  --dark-color: #24292e;
+  --dark-color: #1a1d20;
   --primary-color: #f5f5f5;
```

Comment:
Calling this the primary colors is a bit odd. Usually, the primary color is a color like red or purple, not a shade of gray. Maybe let's rename this to darkgray, lightgray, etc. I know this was something you didn't change but I think it would be nice. 

Let's do this in a follow up. I'll merge for now.

Reply:
Ok, I can fix it 👍

Reply:
```
  --white-color: #ffffff;
  /* --primary-light-color: #ffffff; */

  --light-gray-color: #f5f5f5;
  /* --primary-color: #f5f5f5; */

  --gray-color: #dedede;
  /* --light-color: #dedede; */

  --dark-gray-color: #c0c0c0;
  /* --primary-dark-color: #c0c0c0; */

  --extra-dark-gray-color: #666;
  /* --extra-dark-gray-color: #666; */
```

what about these names? any suggestions for naming.